Shared/Samples/Display clusters/README.md

```diff
@@ -35,6 +35,10 @@ Pan and zoom the map to view how clustering is dynamically updated. Toggle clust
 
 This sample uses a [web map](https://www.arcgis.com/home/item.html?id=8916d50c44c746c1aafae001552bad23) that displays the [Esri Global Power Plants](https://www.arcgis.com/home/item.html?id=eb54b44c65b846cca12914b87b315169) feature layer with feature reduction enabled. When enabled, the cluster's symbology shows the color of the most common power plant type, and a size relative to the average plant capacity of the cluster.
 
+## Additional information
+
+Graphics in a graphics overlay can also be aggregated into clusters. To do this, set the `FeatureReduction` property on the `GraphicsOverlay` to a new `ClusteringFeatureReduction`.
```

Comment:
Should `FeatureReduction` be `featureReduction` or `GraphicsOverlay.featureReduction`? Not sure... 
If not, good to go.